app/models/piece.rb

```diff
@@ -2,6 +2,11 @@ class Piece < ActiveRecord::Base
   belongs_to :user
   belongs_to :game
 
+  def move_piece(new_x, new_y)
```

Comment:
actually, we wouldn't need to update the method.  The current move_to! method currently already updates cords if there are no pieces on the new cords :)